regression/regression/main.nut

```diff
@@ -1358,6 +1358,21 @@ function Regression::Station()
 	}
 }
 
+function Regression::StationList()
+{
+	print("");
+	print("--StationList--");
+	local road_stations = AIStationList(AIStation.STATION_TRUCK_STOP);
+	for (local st = road_stations.Begin(); !road_stations.IsEnd(); st = road_stations.Next()) {
+		print("  GetName()" + AIStation.GetName(st));
```

Comment:
Not sure if we should go for pretty or not, though most seem to add a colon and some spaces.
```suggestion
		print("  GetName(): " + AIStation.GetName(st));
```